app/models/user.rb

```diff
@@ -1,2 +1,6 @@
 class User < ApplicationRecord
+  # scope: active -> (name) { where(name: name) }
+
+  after_create :test_created
+  before_save :test_updated
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/purvak-simform/DemoDangerTool/blob/650dc568f8b1a3f4f40cd63d63336677b4fbfbe5/app/models/user.rb#L5"></span><code>before_save</code> is supposed to appear before <code>after_create</code>.</td>
    </tr>
  </tbody>
</table>